libs/utils/utils.js

```diff
@@ -152,7 +158,9 @@ export const [setConfig, getConfig] = (() => {
       document.documentElement.setAttribute('lang', config.locale.ietf);
       try {
         const contentDir = getMetadata('content-direction');
-        document.documentElement.setAttribute('dir', contentDir || config.locale.dir || (config.locale.ietf ? (new Intl.Locale(config.locale.ietf)?.textInfo?.direction) : null) || 'ltr');
+        const locietf = config.locale.ietf;
+        const ietfDir = locietf ? (new Intl.Locale(locietf)?.textInfo?.direction) : null;
+        document.documentElement.setAttribute('dir', contentDir || config.locale.dir || ietfDir || 'ltr');
```

Comment:
Not a big deal at all, but I think the var names could be better

```suggestion
        const { ietf } = config.locale;
        const intlLocaleDir = ietf && new Intl.Locale(ietf)?.textInfo?.direction;
        document.documentElement.setAttribute('dir', contentDir || config.locale.dir || intlLocaleDir || 'ltr');
```

Another option would be:
```
document.documentElement.setAttribute(
  'dir',
  contentDir
  || config.locale.dir
  || (config.locale.ietf && new Intl.Locale(config.locale.ietf)?.textInfo?.direction)
  || 'ltr',
);
```

Reply:
oops, just saw @auniverseaway comment.  use that, but textInfo needs to use `?.`

Reply:
Sure @chrischrischris I can change the var names or add the shared snippet.
@auniverseaway @chrischrischris Could you please confirm If I should include the snippet or change the variable names?

Reply:
I like this option. It's a lot more scannable and shows the hierarchy of importance what decides the direction.
```js
document.documentElement.setAttribute(
  'dir',
  contentDir
  || config.locale.dir
  || (config.locale.ietf && new Intl.Locale(config.locale.ietf)?.textInfo?.direction)
  || 'ltr',
);
```